com.unity.shadergraph/Editor/Data/Nodes/Procedural/Noise/SimpleNoiseNode.cs

```diff
@@ -48,7 +48,7 @@ public override void GenerateNodeFunction(FunctionRegistry registry, GenerationM
             registry.ProvideFunction("Unity_SimpleNoise_RandomValue_$precision", s => s.Append(@"
 inline $precision Unity_SimpleNoise_RandomValue_$precision ($precision2 uv)
 {
-    return frac(sin(dot(uv, $precision2(12.9898, 78.233)))*43758.5453);
+    return frac(sin(fmod(dot(uv, $precision2(12.9898, 78.233)), TWO_PI))*43758.5453);
```

Comment:
I think what we'll want is a mode dropdown so that we can preserve the existing behavior for anyone who has built projects relying on the behavior (as well as our own tests), as well as add the new mode.

See com.unity.shadergraph/Editor/Data/Nodes/Procedural/Shape/RectangleNode.cs for a simple example of how this was done for another node (git blame the file and look for the changes I made).  Feel free to reach out on Slack if it's not clear.

Reply:
As discussed on slack I made this change platform/API specific